WeatherApp.py

```diff
@@ -0,0 +1,41 @@
+import requests
+from bs4 import BeautifulSoup
+from tkinter import Label
+from tkinter import Tk
+from PIL import ImageTk, Image
+
+url = "https://weather.com/weather/today/l/edaeb599b4628b2e6fb320cbc657a02a651ee03805c8f4ac54528a6804304ec7"
+
+master = Tk()
+master.title("Weather App")
+master.config(bg="white")
+
+img = Image.open("C:/Users/Windows PC/PycharmProjects/learningPython/PartlyCloudy.png")
```

Comment:
This absolute path only works on your machine. Please use a relative path so it works no matter where I choose to clone the repo

Reply:
OK. Sorry about that.  I will fix that tomorrow